config/deploy/staging.rb

```diff
@@ -9,6 +9,19 @@
 # server 'db.example.com', user: 'deploy', roles: %w{db}
 server 'figgy-staging1', user: 'deploy', roles: %w[app db web worker]
 
+before "deploy:assets:precompile", "deploy:npm_install"
```

Comment:
Do we need to do this in production? If so, probably better to move this to `config/deploy.rb`.